src/app/globals.css

```
@@ -1536,3 +1536,53 @@ input[type="range"].appearance-none::-moz-range-track {
+  vertical-align: middle;
+}
+/* One flat class per dot rather than a descendant rule: the setup-ground
+   parity test parses every selector in this file with its own small engine,
+   which reads classes, attributes and a few pseudo-classes but no combinators.
+   The stagger is an inline animation-delay on the second and third dot. */
+.coding-agent-working-dot {
+  display: inline-block;
+  width: 4px;
+  height: 4px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)